docs/reference/search/search.asciidoc

```diff
@@ -302,6 +327,58 @@ If the <<search-api-scroll-query-param,`scroll` parameter>> is specified, this
 value cannot be `0`.
 --
 
+`_source`::
+(Optional)
+Indicates which <<mapping-source-field,source fields>> are returned for matching
+documents. These fields are returned in the `hits._source` property of
+the search response. Defaults to `true`.
++
+.Valid values for `_source`
+[%collapsible%open]
+====
+`true`::
+(boolean)
+The entire document source is returned. This does not include:
+* <<multi-fields,multi-fields>>
+* <<alias,field aliases>>
+* Values copied using the <<copy-to,`copy_to`>> mapping parameter
+
+`false`::
+(boolean)
+The document source is not returned.
+
+`<wildcard_pattern>`::
+(string)
+Wildcard (`*`) patterns containing source fields to return.
```

Comment:
I think this should be singular 'pattern'. Also, what would you think of combining these two 'wildcard' sections for simplicity?